torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -343,11 +343,11 @@ def _is_valid_binary(match, fn):
         if any(
             not (
                 hasattr(n.args[0], "meta")
-                and isinstance(n.args[0].meta.get("val", None), torch.Tensor)
+                and isinstance(n.args[0].meta.get("val", None), torch.Tensor)  # type: ignore[union-attr]
```

Comment:
I guess we should probably type `meta` as a Dict of Any values?

Reply:
SEND FIXES PLZ!!